docs/source/models/concept_models/vivarium_mncnh_portfolio/hemoglobin_component/module_document.rst

```diff
@@ -106,51 +106,46 @@ This module will:
+    - Use same propensity value as decision node #5 and #9 to answer this question
+  * - 3
     - ANC later in pregnancy?
     - As informed from module input (output from :ref:`ANC module <2024_vivarium_mncnh_portfolio_anc_module>`)
     - 
-  * - 5
+  * - 4
     - Hemoglobin screen?
     - Coverage defined by scenario, see :ref:`pregnancy component scenario table <MNCNH pregnancy component scenario table>`. Probability of "yes" is equal to scenario-specific coverage.
     - 
-  * - 6
+  * - 5
     - Receive IFA/MMS *for the first time* at late pregnancy visit?
-    - Coverage defined by scenario, see :ref:`pregnancy component scenario table <MNCNH pregnancy component scenario table>`. If answer to decision node #3 is no, then answer to this decision node is also no. Answer can only be yes if ANC attendance==ANC in later pregnancy ONLY. Then, probability of "yes" is equal to scenario-specific coverage.
-    - Use same propensity value as decision node #1 to answer this question
-  * - 7 
+    - Coverage defined by scenario, see :ref:`pregnancy component scenario table <MNCNH pregnancy component scenario table>`. If answer to decision node #3 is no, then answer to this decision node is also no. Otherwise, probability of "yes" is equal to scenario-specific coverage.
+    - Use same propensity value as decision node #2 and #9 to answer this question
+  * - 6 
     - Hemoglobin screening value <100 g/L? (Based on IFA/MMS adjusted exposure)
     - Instructions detailed on the :ref:`anemia screening intervention page <anemia_screening>`
     - 
-  * - 8
+  * - 7
     - Ferritin screen?
     - Coverage defined by scenario, see :ref:`pregnancy component scenario table <MNCNH pregnancy component scenario table>`. Probability of "yes" is equal to scenario-specific coverage
     - 
-  * - 9
+  * - 8
     - Low ferritin screening value?
     - Instructions detailed on the :ref:`anemia screening intervention page <anemia_screening>`
     - 
-  * - 10
+  * - 9
     - IV iron?
     - Coverage defined by scenario, see :ref:`pregnancy component scenario table <MNCNH pregnancy component scenario table>`. Probability of "yes" is equal to scenario-specific coverage.
     - 
-  * - 11
+  * - 10
     - Also receive IFA/MMS *for the first time* at late pregnancy visit?
-    - Coverage defined by scenario, see :ref:`pregnancy component scenario table <MNCNH intrapartum component scenario table>`. If answer to decision node #3 is no, then answer to this decision node is also no. Answer can only be yes if ANC attendance==ANC in later pregnancy ONLY. Then, probability of "yes" is equal to scenario-specific coverage.
-    - Use same propensity value as decision node #1 to answer this question
+    - Coverage defined by scenario, see :ref:`pregnancy component scenario table <MNCNH intrapartum component scenario table>`. If answer to decision node #3 is no, then answer to this decision node is also no. Otherwise, probability of "yes" is equal to scenario-specific coverage.
```

Comment:
Also wondering about the "Answer can only be yes if ANC attendance==ANC in later pregnancy ONLY." being removed here